src/index.ts

```diff
@@ -14,6 +12,10 @@ addEventListener('fetch', (event: any) => {
 
 function registerRoutes(router: Router) {
     router.get('/', validators);
+    router.get('/active', handlerActive);
+    router.get('/jailed', handlerJailed);
+    router.get('/tombstoned', handlerTombstoned);
+    router.get('/never-jailed', validators);
```

Comment:
This is missing degraded

Reply:
Implemented degraded @ankurdotb . Finalizing the last one now, which is never jailed.